common/interpreter/CMakeLists.txt

```diff
@@ -19,21 +19,24 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
 ################################################################################
 
 if(NOT UNTRUSTED_ONLY)
-	FILE(GLOB PROJECT_HEADERS *.h)
-	FILE(GLOB PROJECT_SOURCES *.cpp)
+    FILE(GLOB PROJECT_HEADERS *.h)
+    FILE(GLOB PROJECT_SOURCES *.cpp)
 
-	PROJECT(pdo_contracts CXX)
+    PROJECT(pdo_contracts CXX)
 
-	ADD_LIBRARY(${PROJECT_NAME} STATIC ${PROJECT_HEADERS} ${PROJECT_SOURCES})
+    ADD_LIBRARY(${PROJECT_NAME} STATIC ${PROJECT_HEADERS} ${PROJECT_SOURCES})
 
-	TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE -fpic -Wno-write-strings -std=c++11)
-	TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-nostdinc")
-	TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-nostdinc++")
-	TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-fno-builtin-printf")
+    TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE -fpic -Wno-write-strings -std=c++11)
+    TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-nostdinc")
+    TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-nostdinc++")
+    TARGET_COMPILE_OPTIONS(${PROJECT_NAME} PRIVATE "-fno-builtin-printf")
 
-	TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include")
-	TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include/tlibc")
-	TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include/libcxx")
+    TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PRIVATE ".." "../state" "../crypto")
+    TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC ${SGX_SSL}/include)
+
+    TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include")
+    TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include/tlibc")
+    TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC "${SGX_SDK}/include/libcxx")
 endif()
```

Comment:
These tab -> spaces changes are correct - they are my fault for accidentally using tabs on my recent PR.